packages/rpc-graphql/src/loaders/transaction.ts

```diff
@@ -30,39 +30,26 @@ function normalizeArgs({ commitment = 'confirmed', encoding = 'jsonParsed', sign
 async function loadTransaction(rpc: Rpc, { signature, ...config }: ReturnType<typeof normalizeArgs>) {
     const { encoding } = config;
 
-    const transaction = await rpc
-        .getTransaction(
-            signature,
-            // @ts-expect-error FIXME: https://github.com/solana-labs/solana-web3.js/issues/1984
-            config,
-        )
-        .send()
-        .catch(e => {
-            throw e;
-        });
```

Comment:
`await` throws errors naturally from this callsite. Did you find something that made it necessary to explicitly supply a `catch` callback @buffalojoec?

Reply:
I don't actually remember specifically why I wrote a `catch` there, but I know some RPC errors weren't showing up in the tests when I was debugging, and I'm almost certain I also has `--verbose` passed to Jest.